packages/suite/src/components/wallet/SearchAction.tsx

```diff
@@ -104,14 +107,14 @@ export const SearchAction = ({
                 innerAddon={
                     <Icon
                         name="search"
-                        cursorPointer
```

Comment:
now I noticed that the icon is clickable when the input is expanded (it folds it), so it should have 
`cursorPointer={isExpanded}`